### ISSUE TITLE
Linter shows failure, and files which failed linting, but is not making comments on the code in the files via of the PR

### ISSUE THREAD

**Issue by Christian-Oleson** [wearerequired/lint-action#608]
Using an ASP.NET Framework (.NET Framework v4.7.2) solution with many projects (51) in the solution, lining fails to show on the files which have changes violating the editorconfig rules. The linter fails, and by clicking the red x in the PR to see the check failure, the annotations show, but on the files tab of the PR, the annotations are missing. An example of this page is:
https://github.com/wearerequired/lint-action/pull/606/**files**

Tool: dotnet format
Action: Runs on windows-latest
Pre-requisites: Runs the following:
- Setup MSBuild
- Restore Packages
- Builds Solution

Runs the linter with the commands:

    - name: Run linters
      uses: wearerequired/lint-action@v2
      with:
          dotnet_format: true
          dotnet_format_args: "./MySolutionName.sln"

**Comment by Christian-Oleson** [comment]
This should not be closed since there is yet to be a response.

**Comment by ocean90** [comment]
Hello there, can you further explain how #606 is related to this? Could this be a duplicate of #577?

**Comment by ocean90** [comment]
What happens when you click on the "Check failure…" link? Looking at https://github.com/wearerequired/lint-action-test/runs/4766567874 and still seems to work.

**Comment by ocean90** [comment]
That's correct, this limitation is forced by GitHub. It's recommended to fix all issues first before using a linter action.

**Comment by Christian-Oleson** [comment]
> That's correct, this limitation is forced by GitHub. It's recommended to fix all issues first before using a linter action.

Right, but even when I fix those errors, it does not show on the files tab.

**Comment by ocean90** [comment]
Do you have an example? https://github.com/wearerequired/lint-action-test/pull/6/files shows them correctly.

**Comment by Christian-Oleson** [comment]
Yup, here is a repo I published here:
https://github.com/Christian-Oleson/lint-action-test

Where the linter ran successfully with errors found here:
https://github.com/Christian-Oleson/lint-action-test/pull/1/checks?check_run_id=12352172533

But the files are not adding annotations on the files tab:
[UnitTest1.cs](https://github.com/Christian-Oleson/lint-action-test/pull/1/files#diff-fe1de19e339ee69306a2b2fe2b0e281d56958dfc28c3e62d61bfb5db9561bd24)

**Comment by Christian-Oleson** [comment]
> Based on the `Check failure on line 7 in lint-action-test\UnitTest1.cs` message I'd say that this is because of the backslash and GitHub seems to expect a forward slash.

Thanks for catching that. I'm not entirely sure I see where it is failing, but I'll trust you've understood it. It sounds like a windows issue for sure though, which is indicative of where I'm see the problem (windows containers).
